framework/src/base/MooseApp.C

```diff
@@ -1525,16 +1525,16 @@ MooseApp::run()
     return;
   }
 
-  try
-  {
-    TIME_SECTION("setup", 2, "Setting Up");
-    setupOptions();
-    runInputFile();
-  }
-  catch (std::exception & err)
-  {
-    mooseError(err.what());
-  }
+  // try
+  // {
+  // TIME_SECTION("setup", 2, "Setting Up");
+  setupOptions();
+  runInputFile();
+  // }
+  // catch (std::exception & err)
+  // {
+  //   mooseError(err.what());
+  // }
```

Comment:
Man, this is a really annoying try that you have to disable for debugging. Once you are in a good shape and you don't need the debugger anymore, make sure you reset this guy.